sys-fs/multipath-tools/multipath-tools-0.6.4.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit systemd toolchain-funcs udev vcs-snapshot
+
+DESCRIPTION="Device mapper target autoconfig"
+HOMEPAGE="http://christophe.varoqui.free.fr/"
+SRC_URI="http://git.opensvc.com/?p=multipath-tools/.git;a=snapshot;h=${PV};sf=tgz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+IUSE="systemd rbd"
+
+RDEPEND=">=sys-fs/lvm2-2.02.45
+	>=virtual/udev-171
+	dev-libs/libaio
+	dev-libs/userspace-rcu
+	sys-libs/readline:0=
+	rbd? ( sys-cluster/ceph )
+	systemd? ( sys-apps/systemd )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.6.2-ignore-modprobe-failures.patch
+
+	# https://www.redhat.com/archives/dm-devel/2017-January/msg00022.html
+	"${FILESDIR}"/${P}-sysmacros.patch
+)
+
+get_systemd_pv() {
+	use systemd && \
+		$(tc-getPKG_CONFIG) --modversion systemd
+}
+
+src_prepare() {
+	default
+
+	# The upstream lacks any way to configure the build at present
+	# and ceph is a huge dependency, so we're using sed to make it
+	# optional until the upstream has a proer configure system
+	if ! use rbd ; then
+		sed -i -e "s/libcheckrbd.so/# libcheckrbd.so/" libmultipath/checkers/Makefile
+		sed -i -e "s/-lrados//" libmultipath/checkers/Makefile
+	fi
+}
+
+src_compile() {
+	# LIBDM_API_FLUSH involves grepping files in /usr/include,
+	# so force the test to go the way we want #411337.
+	emake \
+		LIBDM_API_FLUSH=1 SYSTEMD="$(get_systemd_pv)"
+}
+
+src_install() {
+	dodir /sbin /usr/share/man/man{5,8}
+	emake \
+		DESTDIR="${D}" \
+		SYSTEMD=$(get_systemd_pv) \
+		unitdir="$(systemd_get_systemunitdir)" \
+		libudevdir='${prefix}'/"$(get_udevdir)" \
+		install
+
+	newinitd "${FILESDIR}"/rc-multipathd multipathd
+	newinitd "${FILESDIR}"/multipath.rc multipath
+
```

Comment:
please readd the sed

    sed -i 's|ExecStartPre=/|ExecStartPre=-/|g' "${D}"/usr/lib/systemd/system/multipathd.service || die